internal/pkg/errors/error_message.go

```diff
@@ -145,7 +145,7 @@ const (
 	InvalidAvailableFlagErrorMsg                     = "invalid value \"%s\" for `--availability` flag"
 	InvalidAvailableFlagSuggestions                  = "Allowed values for `--availability` flag are: %s, %s."
 	InvalidTypeFlagErrorMsg                          = "invalid value \"%s\" for `--type` flag"
-	InvalidTypeFlagSuggestions                       = "Allowed values for `--type` flag are: %s, %s, %s."
+	InvalidTypeFlagSuggestions                       = "Allowed values for `--type` flag are: %s, %s, %s, %s."
```

Comment:
Looks like this is only being used in one place, so does it need to be global?

Reply:
Also, can we use one `%s` and insert the entire comma-delimited list so we don't have to keep updating this as more cluster types are added?